ui/desktop/electron-app/config/forge.config.js

```diff
@@ -2,18 +2,9 @@ const process = require('process');
 
 module.exports = {
   hooks: {
-    preMake: (forgeConfig) => {
-      if (process.env.BOUNDARY_DESKTOP_SIGNING_IDENTITY) {
-        forgeConfig.packagerConfig.osxSign = {
-          identity: process.env.BOUNDARY_DESKTOP_SIGNING_IDENTITY,
-          hardenedRuntime: true,
-          "gatekeeper-assess": false,
-          entitlements: "config/macos/entitlements.plist",
-          "entitlements-inherit": "config/macos/entitlements.plist",
-          "signature-flags": "library"
```

Comment:
Fun fact, this forgeConfig update doesn't persist into the following 'make' step. Signing wasn't actually occurring.